contributed_definitions/NXazint2d.nxdl.xml

```diff
@@ -0,0 +1,317 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl" ?>
+<!--
+# NeXus - Neutron and X-ray Common Data Format
+# 
+# Copyright (C) 2008-2022 NeXus International Advisory Committee (NIAC)
+# 
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#
+# For further information, see http://www.nexusformat.org
+-->
+<definition
+    name="NXazint2d"
+    extends="NXobject"
+    type="group"
+    category="application"
+    xmlns="http://definition.nexusformat.org/nxdl/3.1"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd "
+    >
+    <symbols>
+      <doc>
+        The symbol(s) listed here will be used below to coordinate datasets with the same shape.
+      </doc>
+      <symbol name="nImg">
+        <doc>Number of intergated images</doc>
+      </symbol>
+      <symbol name="nRad">
+        <doc>Number of radial bins</doc>
+      </symbol>
+      <symbol name="nRadEdge">
+        <doc>Number of radial bin edges (nRad+1)</doc>
+      </symbol>
+      <symbol name="nEta">
+        <doc>Number of azimuthal bins</doc>
+      </symbol>
+      <symbol name="nEtaEdge">
+        <doc>Number of azimuthal bin edges (nEta+1)</doc>
+      </symbol>
+    </symbols>
+
+    <doc>
+      This application definition describes data from two-dimensional area detectors that has been
+      integrated azimuthally, with a certain radial binning in units of q or 2theta and with
```

Comment:
The data is mapped, projected, binned or resampled to a polar coordinate system - not sure it's strictly an integration!